CONTRIBUTING.md

```diff
@@ -30,20 +31,31 @@ for two kinds of contributors:
 In this guide, we start off by explaining contribution steps that apply
 to both kinds of contributors. Then, the guide branches off to explain specifics for each.
 
+## General guidelines for content on [conda.org][conda-dot-org]
+
+[conda.org][conda-dot-org] is a website which represents the entire conda community. To make sure that not any one project, company or organization
+gets preferential treatment, we have defined the following guidelines to follow when contributing new content and updating
+existing content:
+
+- Content not present in the blog should speak of the wider ecosystem and not prioritize any particular tool, channel, company or organization.
+- The blog is an area where particular tools, channels, companies or organizations may be promoted.
+
 ## Documentation tools
 
-conda.org is built with [Docusaurus 2][docusaurus], a modern static website generator that 
+[conda.org][conda-dot-org] is built with [Docusaurus 2][docusaurus], a modern static website generator that 
 uses a friendly Markdown format to create the website copy. If you are not familiar with 
 Markdown, check out the [Markdown Guide project][markdown-guide].
 
 ## Working on issues
 
-The conda.org project has an [issue board][issue-board] from which technical contributors 
+The conda.org project has an [issue board][issue-board] from which contributors 
 can choose issues. If there is an aspect of the website that you feel needs work, please 
 create a new issue using the **Add item** button at the bottom of the appropriate column.
 
-To be assigned to an existing issue, comment on it and one of the maintainers will 
-ensure you are assigned.
+For new contributors, please look for issues which have the https://github.com/conda-incubator/conda-dot-org/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+label. If you want to work on an issue, please leave a comment saying so. The first person
+to comment on the issue will be assigned, if they are unable to work on it for any reason
+the second person who comments on it will be assigned and so on.
```

Comment:
```suggestion
to comment on the issue will be assigned. If they are unable to work on it for any reason,
the second person who comments on it will be assigned, and so on.
```